code/modules/events/stray_cargo.dm

```diff
@@ -46,23 +81,33 @@
 		var/turf/T = i
 		if(T.density)
 			valid_turfs -= T
-	var/turf/LZ = pick(valid_turfs)
+	var/turf/landing_zone
+	if(admin_override_turf)
+		landing_zone = admin_override_turf
+	else
+		landing_zone = pick(valid_turfs)
 	var/pack_type
-	if(possible_pack_types.len)
-		pack_type = pick(possible_pack_types)
+	if(admin_override_contents)
+		pack_type = admin_override_contents
 	else
-		pack_type = pick(stray_spawnable_supply_packs)
+		if(possible_pack_types.len)
+			pack_type = pick(possible_pack_types)
+		else
+			pack_type = pick(stray_spawnable_supply_packs)
 	var/datum/supply_pack/SP
 	if(ispath(pack_type, /datum/supply_pack))
 		SP = new pack_type
 	else  // treat this as a supply pack id and resolving it with SSshuttle
-		SP = SSshuttle.supply_packs[pack_type]
+		if(admin_override_contents)
+			SP = admin_override_contents //Syndicate crates create a new datum while being customized which will result in this being triggered. Outside of this situation this should never trigger
+		else
+			SP = SSshuttle.supply_packs[pack_type]
 	var/obj/structure/closet/crate/crate = SP.generate(null)
```

Comment:
would be fantastic if you could do the housekeeping and clean out this abbreviation var